tests/unit_tests/test_configuration_refresh.py

```diff
@@ -54,17 +53,6 @@ async def test_config_refresh_empty_file(monkeypatch):
     assert config.DB_PASSWORD == "banana"
 
 
-@fast_forward(20)
-async def test_configuration_refresh(config_service, monkeypatch):
```

Comment:
Why is this deleted?

Reply:
Hmm I think I had added a different configuration and then removed it and this was accidental I will check.